src/components/SongCard/SongCard.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import '../SongCard/SongCard.css'
+
+const SongCard = (props) => {
+  const { title, artist, genres, album_cover } = props;
```

Comment:
You can just destructure right in the function parameters: 

const SongCard = ({ title, artist, genres, album_cover }) => {....}

Reply:
good point, wasn't sure how everyone was feeling on that